src/aind_behavior_video_transformation/filesystem.py

```diff
@@ -0,0 +1,128 @@
+"""foobar"""
```

Comment:
A more informative description of the module can be helpful

Reply:
whoops!